index.bs

```diff
@@ -356,8 +356,8 @@ even slightly or slowly, without affecting your driving direction.
 
 The orientation vector of the [=Orientation Sensor=], can be calculated the following way:
 
-The gravity vector points towards the earth's core when mostly stationary and as long as
-we are not at the poles, we have enough vector length to project that gravity vector onto
+The gravity vector points towards the earth's core when mostly stationary and as long as the
+device is not in free fall, there is enough vector length to project that gravity vector onto
```

Comment:
think we shell mention two things:

- being on poles would affect magnetic field vector
- being in free fall would affect gravity vector